docs/shared/recipes/running-tasks/self-hosted-caching.md

```diff
@@ -44,7 +44,159 @@ To learn more about migrating from custom task runners, [please refer to this de
 
 ## Build Your Own Caching Server
 
-We have [published a new RFC](https://github.com/nrwl/nx/discussions/30548) detailing a custom self-hosted cache based on an OpenAPI specification. This will be available before Nx 21, ensuring a smooth migration path for those who are looking for full control.
+Starting in Nx version 20.8, you can build your own caching server using the OpenAPI specification provided below. This allows you to create a custom remote cache server that fits your specific needs. The server is responsible for managing all aspects of the remote cache, including storage, retrieval, and authentication.
+
+Implementation is left up to you, but the server must adhere to the OpenAPI specification provided below to ensure compatibility with Nx's caching mechanism. The endpoints described below involve the transfer of tar archives which are sent as binary data. It is important to note that the underlying format of that data is subject to change in future versions of Nx, but the OpenAPI specification should remain stable.
+
+As long as your server adheres to the OpenAPI spec, you can implement it in any programming language or framework of your choice.
+
+### Open API Specification
+
+{% tabs %}
+{% tab title="Nx 20.8+" %}
```

Comment:
Its weird to have a tabs component with only one tab.  It looks like the tab component doesn't handle it well either.

![Screenshot 2025-04-09 at 12 56 35 PM](https://github.com/user-attachments/assets/d240e548-e1f9-4ff2-81ea-bde908752ae0)